the-blue-alliance-ios/View Elements/Events/EventRankingTableViewCell.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  RankingTableViewCell.swift
+//  the-blue-alliance-ios
+//
+//  Created by Zach Orr on 5/14/17.
+//  Copyright © 2017 The Blue Alliance. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class EventRankingTableViewCell: UITableViewCell {
+    static let reuseIdentifier = "EventRankingCell"
+    public var ranking: EventRanking? {
+        didSet {
+            guard let ranking = ranking, let team = ranking.team else {
+                return
+            }
+            rankLabel?.text = "Rank \(ranking.rank)"
+            numberLabel?.text = "\(team.teamNumber)"
+            nameLabel?.text = team.nickname ?? team.fallbackNickname
+            rankingPointsLabel?.text = "Avg. \(ranking.qualAverage) Points"
```

Comment:
`qualAverage` should be an optional. It exists for 2015, but not for other years (I believe)